### ISSUE TITLE
Dynamic Lights Shine through walls

### ISSUE THREAD

**Issue by boltactiongaming** [TheGameCreators/GameGuruRepo#118]
This has been a long-standing problem but dynamic lights will not be stopped by walls.  
It makes usage and placement extremely difficult for even an advanced user, let alone a new one.

Thanks

**Comment by tarkus1971** [comment]
Yes this has been a problem for a long time, stick a light outside and it will shine through to the inside of a building or through to upper floors in at bottom floor.

**Comment by Teabone3** [comment]
I noticed this recently when using the EBE.

**Comment by UnknowNomad** [comment]
This is part of this issue
#555 
If 555 will be fixed, this will be too
